src/components/DBSettingsPage/index.jsx

```diff
@@ -348,6 +439,69 @@ class DBSettingsPage extends React.Component {
                 </div>
               )}
               <div className="DBButtons">
+                <div className="DBButtonRow">
+                  <PrimaryButton
+                    label="Update Password"
+                    className="ResetBtn DB-Btn"
+                    onClick={this.showUpdateModal}
+                  />
+                  <div className="buttonText">
+                    Changes or updates database password.
+                  </div>
+                </div>
+                {openUpdateModal && (
+                  <div className="ProjectDeleteModel">
+                    <Modal
+                      showModal={openUpdateModal}
+                      onClickAway={this.hideUpdateModal}
+                    >
+                      <div className="DeleteDatabaseModel">
+                        <div className="DeleteProjectModalUpperSection">
+                          <div className="ModalFormHeading">
+                            <h2>Update database password</h2>
+                          </div>
+                          <div className="InnerModalDescription">
+                            <BlackInputText
```

Comment:
How is the idea of adding a **Confirm Password** input as well?

Reply:
@fel-w kindly elaborate on this

Reply:
@neelxie, something like this:
![image](https://user-images.githubusercontent.com/41950223/126472332-68ef72fb-b7c5-4ae4-a3c5-369061f2b344.png)

Reply:
@fel-w I have worked on the feedback
![Screenshot from 2021-07-21 18-26-31](https://user-images.githubusercontent.com/32802973/126515820-1e81932a-36da-4213-8183-889d14b854fa.png)